Modules/Loadable/Markups/MRML/vtkCurveMeasurementsCalculator.h

```diff
@@ -46,6 +46,14 @@ class VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkCurveMeasurementsCalculator : pub
   vtkCollection* GetMeasurements();
   //@}
 
+  //@{
+  /// This indicates whether the curve should loop back in on itself,
+  /// connecting the last point back to the first point (disabled by default).
+  vtkSetMacro(CurveIsLoop, bool);
+  vtkGetMacro(CurveIsLoop, bool);
```

Comment:
We use "loop" and "closed" terms quite randomly. It would be better to use only one , consistently. Since closed is used in 6 headers files (many of them are public and would be really hard to change), while loop is used in only 2 headers and only vtkCurveGenerator uses it publicly, I think we should stick to ClosedCurve everywhere (and we can change/deprecate the few existing usages of "loop").

Reply:
OK. I was just trying to follow conventions. I can make the terminology more consistent by getting rid of "loop"

Reply:
Yes, the naming was already inconsistent in different classes (mainly due to bringing in curve maker filter from another module), just wanted to make sure that we are converging towards a consistent naming scheme.